nimterop.nimble

```diff
@@ -20,26 +21,36 @@ proc tsoloud() =
   execCmd "nim c -r tests/tsoloud.nim"
   execCmd "nim cpp -r tests/tsoloud.nim"
 
-proc testall() =
+proc buildToast(options: string) =
+  # pending https://github.com/nim-lang/Nim/issues/9513 simplify this
+  let ExeExt2 = when defined(Windows): ".exe" else: ""
+  execCmd(&"nim c -o:build/toast{ExeExt2} {options} nimterop/toast.nim")
```

Comment:
Could use system.ExeExt

Reply:
it's in os.ExeExt and I started w that but 0.19.2 doesn't support `import os`